Build error when using quarkus-tika and quarkus-narayana-jta extensions, provider com.ctc.wstx.stax.WstxInputFactory not found

Build error, provider `com.ctc.wstx.stax.WstxInputFactory` not found when using quarkus-tika and quarkus-narayana-jta extensions.

```
rm -rf fooBar
mvn io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:create \
    -DprojectGroupId=io.quarkus.qe \
    -DprojectArtifactId=fooBar \
    -DprojectVersion=1.0.0-SNAPSHOT \
    -DplatformArtifactId=quarkus-bom \
    -DclassName="io.quarkus.qe.MyResource" \
    -Dextensions="quarkus-tika,quarkus-narayana-jta"
mvn -f fooBar/pom.xml clean verify
```
```
Build failure: Build failed due to errors
	[error]: Build step io.quarkus.narayana.jta.deployment.NarayanaJtaProcessor#build threw an exception: javax.xml.stream.FactoryConfigurationError: Provider com.ctc.wstx.stax.WstxInputFactory not found
	at javax.xml.stream.XMLInputFactory.newInstance(Unknown Source)
	at com.arjuna.common.util.propertyservice.PropertiesFactoryStax.loadFromXML(PropertiesFactoryStax.java:46)
	at com.arjuna.common.util.propertyservice.AbstractPropertiesFactory.loadFromFile(AbstractPropertiesFactory.java:149)
	at com.arjuna.common.util.propertyservice.AbstractPropertiesFactory.getPropertiesFromFile(AbstractPropertiesFactory.java:102)
	at com.arjuna.common.util.propertyservice.AbstractPropertiesFactory.initDefaultProperties(AbstractPropertiesFactory.java:195)
	at com.arjuna.common.util.propertyservice.AbstractPropertiesFactory.getDefaultProperties(AbstractPropertiesFactory.java:62)
	at com.arjuna.common.util.propertyservice.PropertiesFactory.getDefaultProperties(PropertiesFactory.java:48)
	at io.quarkus.narayana.jta.deployment.NarayanaJtaProcessor.build(NarayanaJtaProcessor.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```

Workaround is to add `com.fasterxml.woodstox:woodstox-core` to the deps of the app.

Hello,

I hit this today with 1.3.0. I have TIKA for parsing media files and Hibenate/Panache/JTA for storing meta data. Tests fail with:

`[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.105 s <<< FAILURE! - in com.zemiak.movies.language.LanguageServiceIntegrationTest
[ERROR] allLanguagesCountGreaterThanZero  Time elapsed: 0.009 s  <<< ERROR!
java.lang.RuntimeException: 
java.lang.RuntimeException: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkus.narayana.jta.deployment.NarayanaJtaProcessor#build threw an exception: javax.xml.stream.FactoryConfigurationError: Provider com.ctc.wstx.stax.WstxInputFactory not found
`

You can hit it running "mvn test" on https://github.com/zemiak/movies

--- a/core/runtime/pom.xml
+++ b/core/runtime/pom.xml
@@ -131,6 +131,7 @@
                         <parentFirstArtifact>org.ow2.asm:asm-commons</parentFirstArtifact>
                         <parentFirstArtifact>org.apache.maven:maven-model</parentFirstArtifact>
                         <parentFirstArtifact>org.codehaus.plexus:plexus-utils</parentFirstArtifact>
+                        <parentFirstArtifact>xml-apis:xml-apis</parentFirstArtifact>
                         <!--
                         This is a temporary fix, we need a better way to be able to
                         add these exclusions outside the quarkus-extension.properties file.

